src/redux/categories/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore, combineReducers } from 'redux';
+import bookReducer from '../books/books';
+import checkStatusReducer from './categories';
+
```

Comment:
- [ ] Please make sure to place your `configureStore.js` file directly inside the redux folder to follow the ducks pattern, Also check your imports and exports while making this change. Right now your `configureStore.js` file is in the categories folder.